tests/Server/ServerConfigTest.php

```diff
@@ -155,14 +167,18 @@ public function testAllowsSettingPersistedQueryLoader(): void
     {
         $config = ServerConfig::create();
 
-        $loader = static function (): void {
-        };
-        $config->setPersistedQueryLoader($loader);
-        self::assertSame($loader, $config->getPersistedQueryLoader());
+        $callable = [self::class, 'loadPersistedQuery'];
+        $config->setPersistedQueryLoader($callable);
+        self::assertSame($callable, $config->getPersistedQueryLoader());
+
+        $closure = Closure::fromCallable($callable);
+        $config->setPersistedQueryLoader($closure);
+        self::assertSame($closure, $config->getPersistedQueryLoader());
+    }
 
-        $loader = 'date'; // test for callable
-        $config->setPersistedQueryLoader($loader);
-        self::assertSame($loader, $config->getPersistedQueryLoader());
+    public static function loadPersistedQuery(): ?string
+    {
+        return rand() === 0 ? '{ foo }' : null;
```

Comment:
I'd avoid randomness in tests

Reply:
Right. I was trying to match the signature and figured that was an easy way to make PHPStan not complain about useless code, but now that you mentioned it it is unnecessary.